[[ Community Docs ]] colors - formatting fixes

asayd opened the pull request:
- List of color lines was indented too much, causing them to be formatted as a code block instead of a list.
- Fixed reference to object.
- Hard wrapped long lines.

asayd committed:
[[ Community Docs ]] colors - formatting fixes

- List of color lines was indented too much, causing them to be formatted as a code block instead of a list.
- Fixed reference to object.
- Hard wrapped long lines.

livecodeali commented on docs/dictionary/property/colors.lcdoc:
```diff
@@ -25,30 +25,41 @@ set the colors of this stack to the colors of stack "Home"
+* Line 5: the <topColor> 
+* Line 6: the bottomColor
+* Line 7: the shadowColor
+* Line 8: the focusColor
```
@asayd similarly these should all be links I think.

asayd committed:
Update colors.lcdoc

- Made color properties into links.
- Added color properties to references.
- Added more complete example.

livecodeali commented:
@livecode-vulcan review ok f6110fc

livecode-vulcan commented:
💙 review by @livecodeali ok f6110fc5bfba35e19ad9387043a52d59cdae1aeb

livecode-vulcan commented:
😎 test success f6110fc5bfba35e19ad9387043a52d59cdae1aeb
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2473)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2483)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2466)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2484)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2490)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2494)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2506)
